### DISCUSSION TITLE
essay/2024/04/02/again-english

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#202] [Comments]
# essay/2024/04/02/again-english

28살 때 처음 영어 공부를 해야겠다고 생각했습니다. 코딩을 더 잘하고 싶어서.

https://jeho.page/essay/2024/04/02/again-english.html

**Comment by osangu** [comment]
최근에 저도 파인만 영상 보고나서 삶에 대한 태도가 달라진 것 같습니다.
소파에 앉아서 불에 대해 설명해주시는 표정이 어린아이 같이 행복해보이시더군요.

옛날에는 성적 때문에 공부했었고 하면서도 이거 배우고 나면 멋진 사람이 되어있을까, 스트레스 받았었습니다.

아예 모든 고민과 스트레스에서 해방되지는 못 했지만,
지금은 '적어도 멋진 사람의 근사치는 될 수 있겠다'라는 생각을 합니다.
그래서 그런지 요즘은 배우는 게 새롭고 생각하는 게 재밌더군요.

마음 먹기에 따라 다르다는 걸 확실히 느낍니다.

**Reply by BenjaminKim** [reply]
공감이 갑니다. 배우는 즐거움을 좀 어릴 때 알았으면 좋았을텐데요.
고통스럽기도 하지만.. 이겨내보고 싶습니다. 😀

**Comment by blackturtle2** [comment]
공감합니다.
그런데 항상 작심삼일 된다는게 문제더라구요.
매년 1년을 회고하면, 길어야 2주..하다가 마는게 영어공부더라구요.ㅠㅠ
그래서 저는 올해는 두어달 전부터 말해보카라는 앱을 매일 연속출석해야겠다고 마음 먹고 실행중입니다.
원래 주말에는 스킵해왔는데, 이게 연속성을 떨어지게 하는 장치더라구요.
그래서 주말도 포함해서 꾸역꾸역 게임처럼 매일 하는 중입니다.
올해는 꼭 작심365일하고 싶네요.ㅎㅎ

**Reply by BenjaminKim** [reply]
그러게요. 꾸준히 하는 것이 정말 어려운 것 같습니다.
마음 독하게 먹고 올해 마지막 날 까지 해보려 해요. 함께 힘내시죠!
